nerves_toolchain_ctng/build.sh

```diff
@@ -122,21 +125,23 @@ if [[ $BUILD_OS = "darwin" ]]; then
     WORK_DMG=$WORK_DIR.dmg
     WORK_DMG_VOLNAME=$ARTIFACT_NAME
 
-    export CURSES_LIBS="-L/usr/local/opt/ncurses/lib -lncursesw"
-    CROSSTOOL_LDFLAGS="-L/usr/local/opt/gettext/lib -lintl"
-    CROSSTOOL_CFLAGS="-I/usr/local/opt/gettext/include"
+    export CURSES_LIBS="-L$(brew --prefix ncurses)/lib -lncursesw"
+    CROSSTOOL_LDFLAGS="-L$HOMEBREW_PREFIX/lib -lintl"
+    CROSSTOOL_CFLAGS="-I$HOMEBREW_PREFIX/include"
 
     # Apple provides an old version of Bison that will fail about 20 minutes into the build.
-    export PATH="/usr/local/opt/bison/bin:$PATH"
-    if [[ ! -e /usr/local/opt/bison/bin/bison ]]; then
+    export PATH="$(brew --prefix bison)/bin:$PATH"
```

Comment:
Are bison and grep going to be in a nonstandard spot which is why HOMEBREW_PREFIX won't work here and it needs to be called again?

Reply:
Don't know. The call is free compared to the time it takes to get errors, so I didn't dig into it. I'm quite sure the script could be simplified more.

Reply:
@jjcarstens I feel totally guilty about not wanting to change anything now that it works, and being 99.999% sure that the change that you suggest will totally work. I going to merge, but I know that every time I see this I'll have the urge to clean it up. The next time that I have time and energy, I'll do that.